exercises/simple-linked-list/src/test/java/SimpleLinkedListTest.java

```diff
@@ -0,0 +1,70 @@
+import java.util.NoSuchElementException;
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.Test;
+
+public class SimpleLinkedListTest {
+
+    @Test
+    public void canCreateANewList() {
```

Comment:
Perhaps a stronger initial test is that `aNewListIsEmpty()` and then `assertThat(list.size()).isEqualTo(0)`.